poincare/src/parsing/parser.cpp

```diff
@@ -330,6 +331,15 @@ void Parser::parseLeftSystemParenthesis(Expression & leftHandSide, Token::Type s
   defaultParseLeftParenthesis(true, leftHandSide, stoppingType);
 }
 
+void Parser::parseSingleQuote(Expression & leftHandSide, Token::Type stoppingType) {
+  if (leftHandSide.isUninitialized()) {
+    m_status = Status::Error; // Left-hand side missing
+  } else {
+    leftHandSide = Derivative::Builder(leftHandSide, Symbol::Builder('x'), Symbol::Builder('x'));
```

Comment:
We might need a way to properly determine the variable to derivate instead of hard coding it.

Reply:
By default, even the diff function is added by the toolbox with x as parameter. And how could we do it properly ? (indeed we can get if I remember well the symbol in an expression). What would happen if there's x and y for instance (because having a syntax error isn't good at all :p) ?